src/plugins/unified_search/public/dataview_picker/dataview_list.tsx

```diff
@@ -82,7 +159,64 @@ export function DataViewsList({
             color="transparent"
             paddingSize="s"
           >
-            {search}
+            <EuiFormRow fullWidth>
```

Comment:
@nlatipov can we add this component to a different file and add a unit test?

Reply:
@stratoula  Yes, sure, thank you. Let's do that.